internal/node/node.bzl

```diff
@@ -195,7 +195,12 @@ def _nodejs_binary_impl(ctx, data = [], runfiles = [], expanded_args = []):
 
     # Provide the target name as an environment variable avaiable to all actions for the
     # runfiles helpers to use.
-    env_vars = "export BAZEL_TARGET=%s\n" % ctx.label
+    target_label = str(ctx.label)
+    if target_label.startswith("@@//"):
```

Comment:
Just curious if lstrip could remove all leading @ ?

Reply:
it could. I'm not sure if you'd want to support labels in non-main repos (which would be different from current behavior), so I just focused on the main-repo case.